test-clients/src/main/java/com/hedera/services/bdd/spec/queries/token/HapiTokenNftInfo.java

```diff
@@ -0,0 +1,49 @@
+package com.hedera.services.bdd.spec.queries.token;
+
+/*-
+ * ‌
+ * Hedera Services Test Clients
+ * ​
+ * Copyright (C) 2018 - 2021 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+import com.google.protobuf.ByteString;
+
+import java.util.Optional;
+import java.util.OptionalLong;
+
+public class HapiTokenNftInfo {
```

Comment:
Shouldn't we assert the creation time as-well?

Reply:
I have set it to check for a valid creation time by default. We currently do not have a use case where the creation time **should be** invalid